pkg/supportbundle/supportbundle.go

```diff
@@ -129,7 +129,7 @@ func GetBundleCommand(appSlug string) []string {
 
 	command := []string{
 		"curl https://krew.sh/support-bundle | bash",
-		fmt.Sprintf("kubectl support-bundle %s --redactors=%s\n", GetSpecURI(appSlug), redactors),
+		fmt.Sprintf("kubectl support-bundle --load-cluster-specs --redactors=%s\n", redactors),
 	}
```

Comment:
Redactors are also discovered but the support-bundle spec, but I'm not sure whether KOTS labels them at the moment. I think we can address this in a separate GH issue.